usecases/optimization_paper/tex/optimization_paper.tex

```diff
@@ -0,0 +1,122 @@
+
+
+%%%% Standard Packages
+%%<additional latex packages if required can be included here>
+\input{macros/tex_macros.tex}  % tex commands not variable
+\input{macros/py_macros.tex}   % tex commands created by script
+
+\newcommand{\figures}{../figures/} % scale factor for affinity
+%%%%
+
+%%%%%=============================================================================%%%%
+%%%%  Remarks: This template is provided to aid authors with the preparation
+%%%%  of original research articles intended for submission to journals published 
+%%%%  by Springer Nature. The guidance has been prepared in partnership with 
+%%%%  production teams to conform to Springer Nature technical requirements. 
+%%%%  Editorial and presentation requirements differ among journal portfolios and 
+%%%%  research disciplines. You may find sections in this template are irrelevant 
+%%%%  to your work and are empowered to omit any such section if allowed by the 
+%%%%  journal you intend to submit to. The submission guidelines and policies 
+%%%%  of the journal take precedence. A detailed User Manual is available in the 
+%%%%  template package for technical guidance.
+%%%%%=============================================================================%%%%
+
+\jyear{2022}%
+
+
+\raggedbottom
+%%\unnumbered% uncomment this for unnumbered level heads
+
+\begin{document}
+
+\title[Awesome Article Title]{Awesome Article Title}
+
+%%=============================================================%%
+%% Prefix	-> \pfx{Dr}
+%% GivenName	-> \fnm{Joergen W.}
+%% Particle	-> \spfx{van der} -> surname prefix
+%% FamilyName	-> \sur{Ploeg}
+%% Suffix	-> \sfx{IV}
+%% NatureName	-> \tanm{Poet Laureate} -> Title after name
+%% Degrees	-> \dgr{MSc, PhD}
+%% \author*[1,2]{\pfx{Dr} \fnm{Joergen W.} \spfx{van der} \sur{Ploeg} \sfx{IV} \tanm{Poet Laureate} 
+%%                 \dgr{MSc, PhD}}\email{iauthor@gmail.com}
+%%=============================================================%%
+% TODO: discuss order of authors, do we need to include more?
+
+
+\author*[1]{\fnm{Atul} \sur{Agrawal}}\email{atul.agrawal@tum.de}
+\equalcont{These authors contributed equally to this work.}
+
+\author[2]{\fnm{Erik} \sur{Tamsen}}\email{erik.tamsen@bam.de}
+\equalcont{These authors contributed equally to this work.}
+
+\author[1]{\fnm{Faidon-Stelios} \sur{Koutsourelakis}}\email{p.s.koutsourelakis@tum.de}
+
+\author[2]{\fnm{Jörg F.} \sur{Unger}}\email{joerg.unger@bam.de}
+
+\affil*[1]{\orgdiv{Data-driven Materials Modeling}, \orgname{Technische Universität München}, \orgaddress{\street{Boltzmannstraße 15}, \city{Garching}, \postcode{85748}, \country{Germany}}}
+
+\affil[2]{\orgdiv{Modeling and Simulation}, \orgname{Bundesanstalt für Materialforschung und -prüfung}, \orgaddress{\street{Unter den Eichen 87}, \city{Berlin}, \postcode{12205}, \country{Germany}}}
+
+
+\input{tex_sections/abstract.tex}
+
+\section{Introduction}\label{sec:introduction}
+\input{tex_sections/introduction.tex}
+

```

Comment:
Should we distinguish between the state-of-the-art and an introduction/overview/goals of the paper. I guess we could put both in the introduction, but I don't think the second part will just be 5 lines. It might also be a good idea that we start with the objective part, because that would allow all the other parts to concentrate on these objectives.

Reply:
This is not supposed to be the suggestion for the actual structure. We should have a discussion together about that.